QStreak.xcodeproj/project.pbxproj

```diff
@@ -203,6 +210,16 @@
 			path = QStreakUITests;
 			sourceTree = "<group>";
 		};
+		D5048AF8243F8FAE0029AE0C /* View Record */ = {
```

Comment:
NIT: Could you make sure that this group is alphabetical with the rest of the view controllers group?